### DISCUSSION TITLE
Suddenly getting validationErrors when trying to create a row on a PostGres DB

### DISCUSSION THREAD

**Discussion by VenSlazer** [Budibase/budibase#10114] [Help]
Hey,

I'm suddenly getting the following response when trying to create rows to previously working PostGres DB Tables: 

`{"status":500,"validationErrors":{"id":["can't be blank"]}}`

The error occurs on Automations, as well as when I'm manually trying to create a new row.

When I set up a new table, everything is working fine. But the old tables seem to be broken.

I'm running Budibase in a Docker Container with version 2.4.32 alongside a Postgres Container.

How can I solve this problem?

**Comment by melohagan** [comment]
Hey @VenSlazer 

I think the latest version is **2.4.31**. Do you recall which version you had upgraded from?

**Reply by VenSlazer** [reply]
Hey @melohagan  ok, that's strange, cause the Docker version says that I'm running 2.4.32 => seems like I'm ahead of time ;-) 
<img width="1728" alt="Bildschirm­foto 2023-03-27 um 13 52 13" src="https://user-images.githubusercontent.com/77432946/227934301-bd905aef-319c-4d20-bdba-b4c41fa139b0.png">

The funny thing is, that it worked last week while beeing on the same version. 
I updated from version 2.4.3.

**Reply by melohagan** [reply]
Thanks @VenSlazer 

> when I'm manually trying to create a new row.

To clarify, do you mean when trying to create a new row in the **Data** section of Budibase?

**Reply by VenSlazer** [reply]
It doesn't matter:

From the App:
<img width="1728" alt="Bildschirm­foto 2023-03-27 um 14 47 20" src="https://user-images.githubusercontent.com/77432946/227944365-c4137b76-f744-42d3-9413-71243a17aef1.png">

From the Data section:
<img width="1728" alt="Bildschirm­foto 2023-03-27 um 14 45 17" src="https://user-images.githubusercontent.com/77432946/227944433-5be0e098-6a87-40de-8b50-a8a8a8b3f4af.png">

From an Automation:
<img width="1728" alt="Bildschirm­foto 2023-03-27 um 14 45 58" src="https://user-images.githubusercontent.com/77432946/227944485-a680bede-9a01-43ef-b838-e906b1ba9118.png">

The error occures, no matter what I'm doing. 
When I set up a new table from within budibase on the same Postgres DB, I can create new entries. But the already existing ones seem to be broken.

**Reply by melohagan** [reply]
Thanks @VenSlazer Could you share a screenshot of **Step 2** as well please?

**Reply by melohagan** [reply]
Also you could try editing the **id** column in your table and setting *Required* to false. For example:

![Screenshot 2023-03-28 at 09 43 00](https://user-images.githubusercontent.com/101575380/228180437-e042ea4b-924f-4cb9-b3e2-a6a9ae7f10b0.png)

**Reply by VenSlazer** [reply]
> Thanks @VenSlazer Could you share a screenshot of **Step 2** as well please?

Yes sure, here is a screenshot:
<img width="1728" alt="Bildschirm­foto 2023-03-28 um 11 09 32" src="https://user-images.githubusercontent.com/77432946/228187855-719bf019-4b37-4b85-8580-e94342892e85.png">

But I don't think that this problem is caused by the automation itself.

**Reply by VenSlazer** [reply]
> Also you could try editing the **id** column in your table and setting _Required_ to false. For example:
> 
> ![Screenshot 2023-03-28 at 09 43 00](https://user-images.githubusercontent.com/101575380/228180437-e042ea4b-924f-4cb9-b3e2-a6a9ae7f10b0.png)

That's not possible, cause the id-column is automatically generated.

**Reply by VenSlazer** [reply]
Is it possible, that the auto generated id is not working anymore?

**Comment by oscargws** [comment]
This is happening to myself and multiple other users by the looks of it #10142 @VenSlazer @melohagan 

This started occurring **without** me changing my budibase version in budibase cloud.

**Comment by melohagan** [comment]
Hey @oscargws @VenSlazer 

Could you guys retry this in the cloud environment. Hopefully the issue has been resolved.
